Simcoe/ErrorHandlingOption.swift

```diff
@@ -15,8 +15,8 @@
  */
 public enum ErrorHandlingOption {
 
-    case Suppress
-    case Default
-    case Strict
+    case suppress
+    case `default`
```

Comment:
Hmmm, I'm not sure if I agree. Because it _is_ the default case, to me it makes sense that it would be called `default`